android-sdk/build.gradle

```diff
@@ -50,6 +54,10 @@ dependencies {
     testImplementation 'org.mockito:mockito-core:3.5.13'
     androidTestImplementation 'androidx.test.ext:junit:1.1.2'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
+    testImplementation 'org.mockito:mockito-inline:4.4.0'
+    // Added for mocking android components in unit testing
+    testImplementation 'org.robolectric:robolectric:4.7.3'
```

Comment:
This is a new dependency and test runner. We should discuss as a team and come to an agreement on new dependencies and test practices

Reply:
Reason for adding 2 dependencies:
1. Mockito inline library: used to mock the final or library class. For instance, CheckoutEventLogger class. with core library, it is going to throw as a could not spy final, static methods or class.
2. Robolectric Runner:  It is used for test doubles (standard unit testing framework). We are required to unit test code with the mocking of android components. For instance, to inject Paymentform class in unit test passed android context, to unit test Webview. It simulated android components in JVM without running on a real-time device.
Reference: http://robolectric.org/#:~:text=Robolectric%20handles%20inflation%20of%20views,do%20on%20a%20real%20device.